### ISSUE TITLE
Is there a way to force OpenWebstart to download and use a 32 bit JVM on a 64 bit Windows machine?

### ISSUE THREAD

**Issue by macgibbon** [karakun/OpenWebStart#374]
Most of our users run 64 bit Windows which can run either a 32 bit or 64 bit JVM, it appears that these machines will alway chose a 64 bit JVM.

We have several legacy tools that we would like to deploy as OpenWebstart applications that use legacy SWT code with native methods that are not available in a 64 bit dll. 

I tried the following in the resource tag of the jnlp, but it still tried to run as a 64 bit jvm.

```
<resources>
  <java version="1.8.0.252"  href="https://cdn.azul.com/zulu/bin/zulu8.46.0.19-ca-fx-jdk8.0.252-win_i686.zip" />  
```

Is there a workaround?

**Comment by Jamila891** [comment]
@macgibbon No there is currently not.

**Comment by janakmulani** [comment]
@macgibbon On Windows 64 bit machine, OWS will choose a 64 bit JVM as it uses OS=WIN64 when searching for matching JVMs in OWS JVMManager list. We will try to find a workaround or a fix for this.

**Comment by sclassen** [comment]
I think we have found a possibility but are currently limited in resources. So we cannot give you an estimate when we will find time to implement this.
If you need this urgently you have the chance the sponsor the development. See https://openwebstart.com/support/

**Comment by AndreasEhret** [comment]
the following pull requests 

https://github.com/AdoptOpenJDK/IcedTea-Web/pull/752
https://github.com/karakun/OpenWebStart/pull/390

will provide the possibility to specify a require-32bit attribute in the JNLP file:

<j2se version="1.8*" **require-32bit="true"**/>

**Comment by AndreasEhret** [comment]
will be available with upcoming 1.3.3 release
